api/apps/api/src/modules/export-cleanup/export-cleanup.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { Cron, CronExpression } from '@nestjs/schedule';
+import { InjectEntityManager } from '@nestjs/typeorm';
+import { EntityManager } from 'typeorm';
+import { ExportCleanup } from './export-cleanup';
+import { apiConnections } from '@marxan-api/ormconfig';
+import { AppConfig } from '@marxan-api/utils/config.utils';
+
+const validityIntervalInHours = AppConfig.get<string>(
+  'storage.cloningFileStorage.artifactValidityInHours',
+);
+
+@Injectable()
+export class ExportCleanupService implements ExportCleanup {
+  private readonly logger = new Logger(ExportCleanupService.name);
+  constructor(
+    @InjectEntityManager(apiConnections.default)
+    private readonly apiEntityManager: EntityManager,
+  ) {}
+
+  async identifyExpiredResources() {
+    const expiredProjectExportsIds = await this.apiEntityManager.query(`
+      SELECT e.id FROM exports e
+        WHERE e.resource_kind = 'project' AND 
+        (AGE(NOW(), e.created_at) <  INTERVAL ${validityIntervalInHours} HOUR OR
+        e.resource_id NOT IN (SELECT id FROM published_projects));
```

Comment:
this should be
```suggestion
        e.resource_id IN (SELECT id FROM published_projects));
```
(again apologies for having gotten my query backwards in the task description)